pyk/src/pyk/cli/args.py

```diff
@@ -206,6 +206,7 @@ class KompileOptions(Options):
     bison_lists: bool
     no_exc_wrap: bool
     outer_parsed_json: bool
+    ignore_warnings: list[str] | None
```

Comment:
Yes, I'd go without the `None` type option unless there is a situation where it could actually be set to `None`, which there shouldn't be as far as I know since you have `[]` as the default.